tests/mock_file_ops.h

```diff
@@ -52,6 +52,20 @@ class MockFileOps : public FileOps
     MOCK_CONST_METHOD3(open, void(std::fstream&, const char*, std::ios_base::openmode));
     MOCK_CONST_METHOD1(commit, bool(QSaveFile&));
 
+    MOCK_METHOD(std::unique_ptr<std::ostream>, open_write, (const fs::path& path), (override, const));
```

Comment:
Nice, it's about time we start moving to `MOCK_METHOD`.  We should think about a PR that does that everywhere...